server/src/auth/jwt.strategy.ts

```diff
@@ -0,0 +1,19 @@
+import { ExtractJwt, Strategy } from 'passport-jwt';
+import { PassportStrategy } from '@nestjs/passport';
+import { jwtConstants } from './constants';
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+
+@Injectable()
+export class JwtStrategy extends PassportStrategy(Strategy) {
+    constructor() {
+        super({
+            jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+            ignoreExpiration: false,
+            secretOrKey: jwtConstants.secret,
+        });
+    }
+    async validate(payload: any) {
+        const user = payload.sub
+        return user;
```

Comment:
I think you should add a todo to ensure that we validate that the user exists in the staff directory